app/models/external_work.rb

```diff
@@ -42,6 +42,8 @@ class ExternalWork < ApplicationRecord
   validates :url, presence: true, url_format: true, url_active: true
```

Comment:
Should AO3 [custom url validator](https://github.com/otwcode/otwarchive/blob/7d27b34b8aad7fc980e155217805cde004a5d696/app/validators/url_format_validator.rb) also call `Addressable::URI.heuristic_parse` instead of checking against a regex? Would avoid weird edge cases where the regex is matched but the parsing failed.